### ISSUE TITLE
Dynamic page title

### ISSUE THREAD

**Issue by TimoBbz** [DSD-DBS/capella-collab-manager#208]
The page title should reflect the project or model.

**Comment by MoritzWeber0** [comment]
Which page title do you refer too? The HTML page title tag?

**Comment by TimoBbz** [comment]
Yes this one, it helps to find the page with browser auto-suggest (when you type in the omni-box, it searches through the history with the urls and the titles, for example, a Youtube video will have a url looking like “https://www.youtube.com/watch?v=dQw4w9WgXcQ” and a title looking like “Artist - Title (Music video) - Youtube - Google”, allowing to be found again tying “Artist Title” in the omnibox.)
Setting this title would help users to resume their work even if they close their browser.
